neural_compressor/torch/utils/constants.py

```diff
@@ -52,3 +52,4 @@
 
 
 PT2E_STATIC_QUANT = "pt2e_static_quant"
+PT2E_DYNAMIC_QUANT = "pt2e_dynamic_quant"
```

Comment:
Seems there is no config registered with PT2E_STATIC_QUANT in config.py. How `config` is mapped to `pt2e_static_quant_entry`?

Reply:
Yes, it's a little trick, as we do not want expose two static quant configs to users. 
At `static_quant_entry`, we redispatch the config to `pt2e_static_quant_entry` if ipex if not imported.

https://github.com/intel/neural-compressor/blob/d4b0f0f1e6d85345c3c7eaee5abba0fb904aa4cc/neural_compressor/torch/quantization/algorithm_entry.py#L143-L151